op-node/rollup/derive/batches_test.go

```diff
@@ -539,6 +558,42 @@ func TestValidBatch(t *testing.T) {
 			},
 			Expected: BatchDrop, // dropped because it could have advanced the epoch to B
 		},
+		{
```

Comment:
Haven't looked at all batches tests, but in case you don't, I think it'd be worthwhile to add a test here that covers the happy path when there are no txs in the activation block to confirm this still works as intended.

Reply:
good idea